Gaijinizer patch/CommonEvents_290.txt

```diff
@@ -159,8 +159,9 @@ UWAAAAAAAAA!!#Ron
 > CONTEXT: CommonEvents_501/162/Dialogue
 > CONTEXT: CommonEvents_509/4/Dialogue
 > CONTEXT: CommonEvents_525/163/Dialogue
-\\N[1] is made round up as it is. It was delicious and was
-......#MTLed
+\\N[1] Was swallowed whole just like that.
+
+He was delicious......#Strate
```

Comment:
Is this the descriptive text of Kazuya being eaten by Arachne?

Reply:
It appears to be shared by all the vore filter scenes. This text shows up alongside the vore filter image, it's shared across all scenes because it's in Arachne's file but it also affected Titanoboa's scene.

Reply:
This one we might have to get a bit creative with since translating it directly is a bit awkward. Maybe something like

```Just like that, \\N[1] was swallowed whole as the demon made a delicious morsel out of him.```

Reply:
Done, I left it as two sentences but changed the wording.